### ISSUE TITLE
Seria possível automatizar as traduções para o inglês e, futuramente, outros idiomas?

### ISSUE THREAD

**Issue by lunaperegrina** [lunaperegrina/awesome-bsky#28]

**Comment by JoaquimCassano** [comment]
Opa
pensando em algo usando API do google tradutor


Vou pegar isso pra testar, e depois retorno

**Comment by JoaquimCassano** [comment]
[UPDATE]
Google tradutor é meio tenso, fui de OpenAI mesmo. Com `gpt-4o-mini` não sai caro. Fiz alguns testes [aqui](https://github.com/JoaquimCassano/awesome-bsky), e parece estar funcionando bem. Ele faz as traduções a cada push, e bota no repo. Só seria bom otimizar ele pra só traduzir o que for adicionado, e não o arquivo inteiro, oq economizaria bastante.

**Comment by Camilotk** [comment]
Seria basicamente pegar a diff do commit traduzir as adições e aplicar o patch de mudanças no arquivo traduzido.

**Comment by JoaquimCassano** [comment]
Yup, preciso botar pra ele pegar só a diff, caso a tradução já tenha sido feita uma vez
